### PR TITLE
4.18.1.0: update release notes and upgrade instruction

### PR DESCRIPTION
**Pull request by weizhouapache**

### CODE REVIEW

**Comment by weizhouapache** [comment]
@blueorangutan docbuild

**Comment by weizhouapache** [comment]
@blueorangutan docbuild

**Comment by weizhouapache** [comment]
@blueorangutan docbuild

**Comment by weizhouapache** [comment]
@blueorangutan docbuild

**Comment by weizhouapache** [comment]
@blueorangutan docbuild

**Comment by weizhouapache** [comment]
@blueorangutan docbuild

File: source/releasenotes/about.rst [hunk 1]
```diff
@@ -17,11 +17,11 @@
 What's New in |release|
 =======================
 
-Apache CloudStack |release| is the initial 4.18 LTS release. It has over 300 fixes
-and features since the 4.17.2.0 release.
+Apache CloudStack |release| is a 4.18 LTS minor release with about xxx fixes
```
Comment (rohityadavcloud) [inline source/releasenotes/about.rst:20]: @weizhouapache do you want to fix this? by saying more than 100/200 fixes?

**Review by rohityadavcloud — CHANGES_REQUESTED** [review]
LGTM, some fixes required, then its ready for merging

File: source/releasenotes/about.rst (hunk 1 above)
Comment (weizhouapache) [reply source/releasenotes/about.rst]: updated to "196 fixes"

**Comment by weizhouapache** [comment]
@blueorangutan docbuild

**Review by rohityadavcloud — APPROVED** [review]

**Comment by weizhouapache** [comment]
@blueorangutan docbuild

**Pull request merged**